app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+  def new
+    @spot = Spot.find params[:spot_id]
+    @review = Review.new
+    @ratings = [1, 2, 3, 4, 5]
```

Comment:
It is good that you are defining the allowed values for a rating, but they can probably go someplace better:

```
class Review < AR::Base
  RATING_VALUES = [1, 2, 3, 4, 5]

  validates :rating, inclusion: { in: RATING_VALUES }
end
```

```
Class ReviewsController < AppController
  def new
    @ratings = Review::RATING_VALUES
  end
end
```

Or, you could call `Review::RATING_VALUES` in your view since it is not hitting the database.